cocos/ui/UIEditBox/UIEditBoxImpl-common.h

```diff
@@ -118,6 +120,7 @@ class CC_GUI_DLL EditBoxImplCommon : public EditBoxImpl
     virtual void setNativeInputMode(EditBox::InputMode inputMode) = 0;
     virtual void setNativeInputFlag(EditBox::InputFlag inputFlag) = 0;
     virtual void setNativeReturnType(EditBox::KeyboardReturnType returnType) = 0;
+    virtual void setNativeKeyboardAppearance(EditBox::KeyboardAppearance) {}
```

Comment:
shouldn't empty realisation, suggest keep pure virtual function, add empty realize function for other platforms, and comment didn't finish. I think it's better to add assertion so that  assertion error occurs when a android developer using this method and try to compile.